src/core/agent/services/identifier.types.ts

```diff
@@ -16,15 +15,7 @@ interface IdentifierShortDetails {
   isPending: boolean;
 }
 
-type GetIdentifierResult =
-  | { type: IdentifierType.KERI; result: KERIDetails }
-  | { type: IdentifierType.KEY; result: DIDDetails };
-
-interface DIDDetails extends IdentifierShortDetails {
-  keyType: string;
-  controller: string;
-  publicKeyBase58: string;
-}
+type GetIdentifierResult = { type: IdentifierType.KERI; result: KERIDetails };
```

Comment:
This type should be removed and we should just have an interface called `IdentifierDetails` which is what `KERIDetails` was